mobile/cafe.go

```diff
@@ -7,12 +7,12 @@ import (
 )
 
 // RegisterCafe calls core RegisterCafe
-func (m *Mobile) RegisterCafe(host string) error {
+func (m *Mobile) RegisterCafe(host string, token string) error {
```

Comment:
Obviously will require modifications to the bridges. Also for our client libs, might be good to have default env-based tokens that we add?